completions/fish/eza.fish

```diff
@@ -1,6 +1,6 @@
 # Meta-stuff
 complete -c eza -s v -l version -d "Show version of eza"
-complete -c eza -s '?' -l help -d "Show list of command-line options"
+complete -c eza -l help -d "Show list of command-line options"
```

Comment:
just FYI `-h` will be assigned to help when #197 will be introduced as it is an industry standard

Reply:
Oops - I missed that. I’ll make a correction.